src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -79,22 +93,45 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
   } = useForm({
     resolver: yupResolver(schema),
     defaultValues: {
-      wbsNum,
       what: crDesc,
       why: [{ type: ChangeRequestReason.Other, explain: '' }],
       type: ChangeRequestType.Issue
     }
   });
   const { fields: whys, append: appendWhy, remove: removeWhy } = useFieldArray({ control, name: 'why' });
-
-  const theme = useTheme();
+  const { isLoading, isError, error, data: projects } = useAllProjects();
 
   const style = { border: '1px solid ' + theme.palette.divider, borderRadius: 2 };
 
   const permittedTypes = Object.values(ChangeRequestType).filter(
     (t) => t !== ChangeRequestType.Activation && t !== ChangeRequestType.StageGate
   );
 
+  if (isLoading || !projects) return <LoadingIndicator />;
+  if (isError) return <ErrorPage message={error?.message} />;
+
+  const wbsDropdownOptions: { label: string; id: string }[] = [];
+  projects.forEach((project: Project) => {
+    wbsDropdownOptions.push({ label: `${wbsPipe(project.wbsNum)} - ${project.name}`, id: wbsPipe(project.wbsNum) });
+    project.workPackages.forEach((workPackage: WorkPackage) => {
+      wbsDropdownOptions.push({
+        label: `${wbsPipe(workPackage.wbsNum)} - ${workPackage.name}`,
+        id: wbsPipe(workPackage.wbsNum)
+      });
+    });
+  });
```

Comment:
Could we display projects and workpackages differently? (might be a product question). But like put - project or - wp next to them or bold projects or smthn. I know you can tell from the wbsNum but it's not super obvious visually.

Reply:
hmmm i don't think i can selectively bold stuff

imo this looks bad

<img width="873" alt="Screenshot 2023-01-08 at 3 13 30 PM" src="https://user-images.githubusercontent.com/63663597/211216942-7fe4d0f2-9139-4017-909d-d16c368dd95b.png">

Reply:
Aight, it's already way better than what we have rn, so we can look into this for a future ticket